### ISSUE TITLE
Storage Name Checking Script

### ISSUE THREAD

**Issue by AurevoirXavier** [darwinia-network/darwinia-common#314]
Add to CI

**Comment by wuminzhe** [comment]
```ruby
excludes = %w[
  support/src/lib.rs
  rpc/runtime-api/src/lib.rs
  rpc/src/lib.rs
]
Dir["/Users/itering/Projects/darwinia-network/darwinia-common/frame/**/*"].each do |d|
  if d.end_with?("src/lib.rs") && excludes.all? { |e| not d.end_with?(e) }
    p d
    File.open(d).read.each_line do |line|
      scaned = line.scan /trait Store for Module.* as(.+){/
      if scaned.length > 0
        puts "--------------------------------" 
        puts scaned[0][0].strip
      end
      
      scaned = line.scan /\t{2}pub ([A-Z]\S*)\s*$/
      puts "    #{scaned[0][0].strip}" if scaned.length > 0

      scaned = line.scan /\t{2}pub ([A-Z]\S*)\s* /
      puts "    #{scaned[0][0].strip}" if scaned.length > 0
    end
    puts ""

  end
end
```

**Comment by HackFisher** [comment]
https://github.com/paritytech/substrate/pull/8123 

https://github.com/paritytech/substrate/pull/8038

**Comment by HackFisher** [comment]
Fixed in #733 ?
